packages/code-du-travail-data/dataset/fiches_service_public/parseReference.js

```diff
@@ -1,8 +1,8 @@
 // Do we really need this one ?
 const find = require("unist-util-find");
 const queryString = require("query-string");
-const cdt = require("../code_du_travail/code-du-travail.json");
-const conventions = require("../conventions-collectives/ccn-list.json");
+const cdt = require("@socialgouv/legi-data/data/LEGITEXT000006072050.json");
```

Comment:
does the parseRef should be adressed in another PR ?

Reply:
can be